### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by JasonKAEC**
Pull request for first version of Components documentation.

### CODE REVIEW

File: src/components/NavigationPopupMenu/NavigationPopupMenu.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ import useLockBodyScroll from './../../utils/useLockBodyScroll'
 
 import LoaderSpinner from './../LoaderSpinner'
 
+/**
+ * Component that renders the NavigationPopupMenu component.
+ *
+ * @component
+ *
+ * @param {} loggedIn - Parameter is not used within this component.
+ * @param {booelan} showBanner - Parameter that if set to true, will show the banner.
```
Comment (Aidenbuis) [inline src/components/NavigationPopupMenu/NavigationPopupMenu.js:26]: Small spelling mistake

File: src/components/NavigationPopupMenu/NavigationPopupMenu.js [hunk 2]
```diff
@@ -17,6 +17,17 @@ import useLockBodyScroll from './../../utils/useLockBodyScroll'
...
+ * @param {} logout - Parameter is not used within this component.
```
Comment (Aidenbuis) [inline src/components/NavigationPopupMenu/NavigationPopupMenu.js:27]: Good catch, feel free to remove the parameter and this doc line to clean it up :)

File: src/components/PopUpStatusAanpassen/PopUpStatusAanpassen.js [hunk 3]
```diff
@@ -6,6 +6,18 @@ import PopUpAnimatedContainer from './../PopUpAnimatedContainer'
...
 import VOLGENDE_STATUS from './../../constants/beleidskeuzeStatusAanpassen'
 
+/**
+ * Component that renders the PopUpStatusAanpassen component.
+ *
+ * @component
+ *
+ * @param {object} dataObject - Parameter that can be set to the patchStatus object if the onclick function is called and selectValue is not empty.
+ * @param {string} fieldValue - Parameter that sets the value element to the select element.
+ * @param {} handleChange - Parameter that is not used within this component.
```
Comment (Aidenbuis) [inline src/components/PopUpStatusAanpassen/PopUpStatusAanpassen.js:16]: Feel free to remove unused parameters :)

**Review by Aidenbuis — APPROVED** [review]
Mooie nieuwe batch aan Docs, goed bezig! 🚀

File: src/components/NavigationPopupMenu/NavigationPopupMenu.js (hunk 2 above)
Comment (JasonKAEC) [reply src/components/NavigationPopupMenu/NavigationPopupMenu.js]: Thanks, will remove the parameters from now on.

File: src/components/NavigationPopupMenu/NavigationPopupMenu.js (hunk 1 above)
Comment (JasonKAEC) [reply src/components/NavigationPopupMenu/NavigationPopupMenu.js]: I see indeed haha, I've fixed the spelling mistake in my last commit.

**Pull request merged**